### PR TITLE
Feature/jms implementation

### PR DESCRIPTION
**Pull request by jbarbaso**
Added JsonB API dependencies to pom file.
Changed Timestamp fields to LocalDateTime type in entity Car and files affected.
Added message producer and message listener to send and read messages from "jms/CarsAppQueue".
Added a new resource to send messages to the queue with REST

### CODE REVIEW

File: src/main/java/com/everis/cars/entity/LocalDateTimeAdapter.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package com.everis.cars.entity;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import javax.xml.bind.annotation.adapters.XmlAdapter;
+
```
Comment (javieraviles) [inline src/main/java/com/everis/cars/entity/LocalDateTimeAdapter.java:7]: would be nice to document whats the end goal of having this class

File: src/main/java/com/everis/cars/jms/CreateCarMessageListener.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+import com.everis.cars.entity.Car;
+
+/**
+ * {@link MessageListener} implementation to listen the cars queue and create,
+ * update or delete received cars from queue
+ * 
+ * @see javax.ejb.MessageDriven
+ */
+@MessageDriven(mappedName = "jms/CarsAppQueue")
+public class CreateCarMessageListener implements MessageListener {
```
Comment (javieraviles) [inline src/main/java/com/everis/cars/jms/CreateCarMessageListener.java]: is there a unit test for this class?

File: src/main/java/com/everis/cars/jms/CreateCarMessageProducer.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+import javax.jms.JMSProducer;
+import javax.jms.Queue;
+
+import org.apache.log4j.Logger;
+
+/**
+ * Producer helper class to connect and send messages to a queue
+ */
+@Stateless
+public class CreateCarMessageProducer {
```
Comment (javieraviles) [inline src/main/java/com/everis/cars/jms/CreateCarMessageProducer.java]: is there a unit test for this class?

**Review by javieraviles — CHANGES_REQUESTED** [review]
Looks neat

File: src/main/java/com/everis/cars/jms/CreateCarMessageListener.java (hunk 2 above)
Comment (jbarbaso) [reply src/main/java/com/everis/cars/jms/CreateCarMessageListener.java]: I refactorized the class and created the test

File: src/main/java/com/everis/cars/jms/CreateCarMessageProducer.java (hunk 3 above)
Comment (jbarbaso) [reply src/main/java/com/everis/cars/jms/CreateCarMessageProducer.java]: Unit test created

**Review by javieraviles — APPROVED** [review]

**Pull request merged**